dts/xtensa/espressif/esp32/esp32_common.dtsi

```diff
@@ -425,5 +425,23 @@
 			status = "disabled";
 		};
 
+		sdhc: sdhc@3ff68000 {
+			compatible = "espressif,esp32-sdhc";
+			reg = <0x3ff68000 0x1000>;
+			interrupts = <SDIO_HOST_INTR_SOURCE>;
+			interrupt-parent = <&intc>;
+			clocks = <&rtc ESP32_SDMMC_MODULE>;
+			status = "disabled";
+
+			slots {
+				#address-cells = <1>;
+				#size-cells = <0>;
+
+				slot1: sdhc-slot@1 {
```

Comment:
I guess you need to add slot0 as well? Otherwise that won't ever be available for application/board.